deps/graph-parser/src/logseq/graph_parser/cli.cljs

```diff
@@ -26,9 +26,11 @@ TODO: Fail fast when process exits 1"
   "Given a git graph directory, returns allowed file paths and their contents in
   preparation for parsing"
   [dir]
-  (let [files (->> (str (.-stdout (sh ["git" "ls-files"]
+  ;; -z needed to avoid quoting unusual paths that cause slurp failures.
+  ;; See https://git-scm.com/docs/git-ls-files#_output for more
+  (let [files (->> (str (.-stdout (sh ["git" "ls-files" "-z"]
```

Comment:
A bug exists before this PR is that non-git graph directory can't be published.

Reply:
Good point. It also effects all graph-parser scripts. I can do this in a follow up PR. To do this I will need to reuse what logseq does to fetch valid files for a graph, which I think is [electron-handler/get-files](https://github.com/logseq/logseq/blob/9bf867e9ffda0cda3c4e3f449fdbb1943e69eea3/src/electron/electron/handler.cljs#L159). Is it ok if I pull it and the code it depends on to deps/common so I can reuse it for electron and the graph-parser?

Reply:
Good idea!

Reply:
Cool. Created https://linear.app/logseq/issue/LOG-2499/graph-parsercli-should-support-non-git-graphs. Plan to knock it out next cycle